pkg/rancher-desktop/components/MarketplaceCatalog.vue

```diff
@@ -85,7 +85,7 @@ export default (Vue as VueConstructor<Vue & VuexBindings>).extend({
         });
       }
 
-      return tempExtensions.filter(item => this.isMobyActive || item.containerd_compatible);
+      return tempExtensions.filter(item => this.isMobyActive ? item.moby_compatible : item.containerd_compatible);
```

Comment:
Enforcing `never` for eslint [multiline-ternary](https://github.com/rancher-sandbox/rancher-desktop/blob/afd118e8a658e8ee35e98715b840a4eb0aef094d/.eslintrc.js#L56) makes wrapping this line a little difficult.

Reply:
> Consider replacing item.moby_compatible with item.moby_compatible ?? false

Can you expand on why the falsy equivalent doesn't suffice in this scenario?

Reply:
Sorry, I'm an idiot. I meant `?? true`, so that we can leave off the places where we set it to true (because most extensions would be moby-compatible).

Reply:
Thanks, that makes more sense 🙂

Are you suggesting that we might also want to remove all instances of `moby_compatible: true` in `_demo_marketplace_items.js` or is it simply that we want to assume moby compatibility in the event that the property might not exist?

Reply:
Right, because this PR adds a bunch of `moby_compatible: true` to `_demo_marketplace_items.js`, so I figured we should make that the default.

But this is all trivial and doesn't _really_ need to be done :)